distribution/docker/solrcloud/pom.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>ddf.distribution</groupId>
+        <artifactId>docker</artifactId>
+        <version>2.26.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>ddf-solrcloud</artifactId>
+    <groupId>ddf.distribution.docker</groupId>
+    <packaging>pom</packaging>
+    <name>DDF :: Distribution :: Docker :: SolrCloud</name>
+
+    <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources-filtered</directory>
+                <filtering>true</filtering>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>false</filtering>
+            </resource>
+        </resources>
+        <plugins>
+            <plugin>
+                <artifactId>maven-clean-plugin</artifactId>
+                <version>3.1.0</version>
+                <configuration>
+                    <!-- avoid directory clean failures on Windows due to exec command holding onto directory lock -->
+                    <failOnError>false</failOnError>
```

Comment:
❓ What are the implications of this? Just force quit some orphaned docker process and try again? If this doesn't fail, how will I know I orphaned something or am otherwise stuck in an invalid development state? 

If the cleanup still occurs regardless of failure, then 👍

Reply:
Only the directory is not deleted.  All files in the classes directory is deleted.  I does retry multiple times before continuing.  This is because of how Windows creates a lock on the directory when executing docker compose from that directory.  Without this you would have to stop your SolrCloud every time you do a build of this module on Windows.

This is the suggested solution on the Maven Clean Plugin FAQ.

http://maven.apache.org/plugins/maven-clean-plugin/faq.html